bastion/gui_qt.py

```
@@ -1306,7 +1306,18 @@ def _launch_gufw_process(self):
                    f'WAYLAND_DISPLAY={wayland_display}',
                    f'XDG_RUNTIME_DIR={xdg_runtime}',
                    'gufw']
-            subprocess.Popen(cmd, start_new_session=True)
+            proc = subprocess.Popen(cmd, start_new_session=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)